script/Ciel/Day 9/09_01_CIEL09_1.txt

```diff
@@ -431,12 +475,12 @@ She's right.
 [sha:5635b2f3bac11af3e5bbb5d59954166522c45aaf]{
 -- Page 34, Offset 22224.
 -- 俺は人前で眼鏡を外すこと―――裸眼で人を視ることを頑なに避けてきた。
-Taking my glasses off in front of other people, in other words, %{i}seeing%{/i} people with my naked eyes, is something I have come to adamantly avoid.
+I normally make it a rule never to take my glasses off around people, so that I don't have to %{i}see%{/i} them.
```

Comment:
to never take

Reply:
mild pref for current version but could go for yusuke's version too tbh